helmfile.d/upstream/grafana/grafana/README.md

```diff
@@ -223,7 +224,7 @@ need to instead set `global.imageRegistry`.
 | `admin.existingSecret`                    | The name of an existing secret containing the admin credentials (can be templated). | `""`                                 |
 | `admin.userKey`                           | The key in the existing admin secret containing the username. | `"admin-user"`                          |
 | `admin.passwordKey`                       | The key in the existing admin secret containing the password. | `"admin-password"`                      |
-| `serviceAccount.autoMount`                | Automount the service account token in the pod| `true`                                                  |
+| `serviceAccount.automountServiceAccountToken` | Automount the service account token on all pods where is service account is used | `false` |
```

Comment:
Does this change in default affect us in any way?

Reply:
not from what I was able to test

Reply:
I'm guessing the datasource/dashboard reloader uses the token to query the k8s api for changes. This still works as intended?

Reply:
Seems to have been a lot of fuss about the change at least :smile: 
* https://github.com/grafana/helm-charts/pull/2977

Reply:
The changes mentioned in this PR are already implemented in the chart version 7.3.7
I did make some changes to a dashboard and added a new datasource. No errors in the logs and the changes were visible in the UI as expected.